lib/phases/reboot.js

```diff
@@ -63,6 +63,7 @@ Reboot.prototype.reboot = function() {
       return phase._restart();
     })
     .then(function(time) {
+      phase.setup();
```

Comment:
'adb forward' is done after the 'adb log' / posting the start mark now, so I'm assuming the order doesn't matter...